src/main/java/frc/robot/RobotContainer.java

```diff
@@ -59,20 +69,33 @@ private void configureBindings() {
       m_Drive.drive(forwardSpeed, rightSpeed, rotatinalSpeed);
     }, m_Drive));
 
-    // Schedule `exampleMethodCommand` when the Xbox controller's B button is pressed,
-    // cancelling on release.
-    //m_driverController.b().whileTrue(m_exampleSubsystem.exampleMethodCommand());
-  }
+    contrJoystick.button(1).onFalse(Commands.run(() -> {
+      m_LaunchMech.launchSpeaker();
+    }, m_LaunchMech));
 
-  /**
-   * Use this to pass the autonomous command to the main {@link Robot} class.
-   *
-   * @return the command to run in autonomous
-   */
-  public Command getAutonomousCommand() {
-    // An example command will be run in autonomous
-    return Autos.exampleAuto(m_exampleSubsystem);
-  }
+    contrJoystick.button(2).onFalse(Commands.run(() -> {
+      m_LaunchMech.launchAmp();
+    },m_LaunchMech));
+
+    contrJoystick.button(3).whileTrue(Commands.run(() -> {
+      m_Intakemech.intake();
+    },m_Intakemech));
+
+    contrJoystick.button(3).onFalse(Commands.run(() -> {
+      m_TrapScoreSubsystem.startAngle();
+    },m_TrapScoreSubsystem));
 
+    contrJoystick.button(4).onFalse(Commands.run(() -> {
+      m_TrapScoreSubsystem.loadAngle();
+    },m_TrapScoreSubsystem));
+
+    contrJoystick.button(5).onFalse(Commands.run(() -> {
+      m_TrapScoreSubsystem.dropAngle();
+    },m_TrapScoreSubsystem));
```

Comment:
same as above:

- onFalse vs whileFalse
- onTrue vs whileTrue
- Commands.run vs Commands.startEnd

but don't let this block you.

Reply:
#32